pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/ClustersBase.java

```diff
@@ -595,6 +595,9 @@ public NamespaceIsolationData getNamespaceIsolationPolicy(
                             brokerIsolationData.namespaceRegex = Lists.newArrayList();
                         }
                         brokerIsolationData.namespaceRegex.addAll(policyData.namespaces);
+                        if (nsPolicyImpl.isPrimaryBroker(broker)) {
+                            brokerIsolationData.isPrimary = true;
+                        }
```

Comment:
Do we need to add the initialization of `isSecondary`?

Reply:
👍

Reply:
Oh, I see, there is no such an area for `isSecondary`
```
public class BrokerNamespaceIsolationData {

    @ApiModelProperty(
        name = "brokerName",
        value = "The broker name",
        example = "broker1:8080"
    )
    public String brokerName;
    @ApiModelProperty(
            name = "policyName",
            value = "Policy name",
            example = "my-policy"
        )
    public String policyName;
    @ApiModelProperty(
            name = "isPrimary",
            value = "Is Primary broker",
            example = "true/false"
        )
    public boolean isPrimary;
    @ApiModelProperty(
        name = "namespaceRegex",
        value = "The namespace-isolation policies attached to this broker"
    )
    public List<String> namespaceRegex;
```

Reply:
Oh, I mean, do we need to add a field `isSecondary` to this entity? @jiazhai